### DISCUSSION TITLE
Not able to login using OTP

### DISCUSSION THREAD

**Discussion by harshalpere** [community/community#116328] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My id is **harshalsph** which is not able to login
Not able to login to account getting error message while verification of OTP
![image (1)](https://github.com/community/community/assets/57993688/58956bd0-f5c5-4f48-8aa2-6cbfb2d1ff98)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @harshalpere! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

I've gone ahead and moved this to the correct category for you. Good luck!
